CMakeLists.txt

```diff
@@ -69,7 +69,7 @@ ENDIF ()
 
 MESSAGE("Install will copy files to ${D3_GAMEDIR}")
 
-include_directories("lib" "Descent3" ${PLATFORM_INCLUDES})
+include_directories("${CMAKE_SOURCE_DIR}" "lib" "Descent3" ${PLATFORM_INCLUDES})

```

Comment:
This is not a great idea to include the root source dir. If you made this change to find cfile.h, then include `cfile` directory directly

Reply:
I agree here, but not sure why this comment was marked as resolved?